src/hooks/api/ownerships.ts

```diff
@@ -50,15 +64,37 @@ const useRequestOwnershipMutation = (configuration: UseQueryOptions = {}) =>
     mutationFn: requestOwnership,
     mutationKey: 'ownerships-request-ownership',
     ...configuration,
-  });
-
-const getOwnershipRequests = async ({ headers, organizerId, ownerId }) => {
+  }) as UseMutationResult<void, Error, RequestOwnershipArguments>;
+
+const getOwnershipRequests = async ({
+  headers,
+  itemId,
+  ownerId,
+  state,
+  paginationOptions,
+}: {
+  headers: Headers;
+  itemId?: string;
+  ownerId?: string;
+  state?: OwnershipState;
+} & PaginationOptions) => {
+  const searchParams = new URLSearchParams();
+  if (paginationOptions) {
+    searchParams.set('limit', `${paginationOptions.limit}`);
+    searchParams.set('offset', `${paginationOptions.start}`);
```

Comment:
Are the template strings to cast to string or something?

Reply:
They are needed because searchParams is a `Record<string, string>` and paginationOptions are passed as numbers